opm/autodiff/MissingFeatures.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+  Copyright 2016 Statoil ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include <opm/common/OpmLog/OpmLog.hpp>
+#include <opm/parser/eclipse/Deck/DeckKeyword.hpp>
+#include <opm/parser/eclipse/Parser/Parser.hpp>
+#include <opm/parser/eclipse/EclipseState/checkDeck.hpp>
+#include <opm/autodiff/MissingFeatures.hpp>
+#include <vector>
+#include <string>
+
+
+namespace Opm {
+
+    
+    void checkKeywords(DeckConstPtr deck, ParserConstPtr parser)
+    {
+        // These keywords are supported by opm-parser, but are not supported
+        // by flow. For some of them, only part of the options are supported.
+        // The list is used to output messages only.
+        std::vector<std::string> unsupported_keywords = {
+            "ACTDIMS", "ADSALNOD", "API", "AQUCON", "AQUDIMS", "AQUNUM"
```

Comment:
Some of the keywords in the list are used by the parser to create a `EclipseState` and not relevant for flow. So - in my opinion the following keywords should be removed from the list:

1. `BLOCK_PROBE, FILED_PROBE, GROUP_PROBE, PERFORMACE_PROBE, REGION_PROBE, WELL_PROBE`. These are regular expression based "catch all" keywords for keywords in the summary section, the `ENDPOINT_SPECIFIERS` keyword is also a list of other keywords.

2. `ACTDIMS`, `AQUDIMS, REGDIMS, SMRYDIMS, UDADIMS, UDQDIMS, PIMTDIMS` - the `xxxxDIMS` keyword originate in the FORTRAN77 memory model with fixed size arrays. The parser uses many of these keywords - typically for consistency checks, but the should *not* pass to flow! The `MEMORY` keyword is also such a very old keyword which is even deprecated in Eclipse.

3. The `ENDINC` keyword will stop parsing an include file - fully supported in the parser.

4. `WRFT, WRFTPLT, UNIFIN, UNIFOUT, FMTIN, RPTONLY, RPTRST, RPTSOL, RPTSCHED`- these are IO related keywords which are partly/fully supported by parser/output - should not extend to `flow`.

There might be other keywords which should be removed from the list as well, maybe it is better to make this list *the other way around* - i.e. starting with *features* we know we do not support, and then make a list of the keywords which configure these features?